client/src/components/viewers/types.ts

```diff
@@ -0,0 +1,18 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { Translatable } from 'megashark-lib';
+
+export interface FileControlsDropdownItemContent {
+  id: number;
```

Comment:
The `id` makes defining the items a bit more annoying, is there really a use for it?